tripy/tests/performance/test_perf.py

```diff
@@ -115,15 +132,13 @@ def func({arg_str}):
         for input in inputs:
             input.eval()
 
-        for _ in range(warm_up_runs):
-            compiled_one_io(*inputs)
+        def measure_thunk():
+            return compiled_one_io(*inputs)
 
-        start = time.perf_counter_ns()
-        for _ in range(iterations):
+        for _ in range(warm_up_runs):
             compiled_one_io(*inputs)
```

Comment:
We can drop the warm-up loop too right? It's already being done in the helper.

Reply:
Done.